pkg/drivers/vfkit/driver_darwin.go

```diff
@@ -442,3 +466,11 @@ func (d *Driver) sendSignal(s syscall.Signal) error {
 
 	return proc.SendSignal(s)
 }
+
+func (d *Driver) supportsVirtiofs() bool {
+	macosVersion, _, err := crcos.RunWithDefaultLocale("sw_vers", "-productVersion")
+	if err != nil {
+		return false
+	}
+	return strings.HasPrefix(macosVersion, "12.")
```

Comment:
maybe return true also for macOS 12+ versions? otherwise we'll have to remember to change it when macOS 13 is released

Reply:
Ah thanks, initially wanted to use semver for this, then went with a shortcut. But yeah, definitely not very future proof this way :)